app/src/main/res/values-es/strings.xml

```diff
@@ -390,6 +390,17 @@
     <string name="settingsAiEngineDescription">Seleccione su asistente de redacción</string>
     <string name="settingsAppLock">Bloqueo de aplicaciones</string>
     <string name="settingsAuthorizeTracking">Autorizar el seguimiento</string>
+    <string name="settingsAutoAdvanceDescription">Seleccione qué mostrar cuando se archiva o elimina un mensaje</string>
+    <string name="settingsAutoAdvanceFollowingThreadDescription">Mostrar el siguiente hilo al archivar o borrar\n</string>
+    <string name="settingsAutoAdvanceFollowingThreadTitle">Siguiendo el hilo</string>
+    <string name="settingsAutoAdvanceListOfThreadsDescription">Mostrar lista de hilos al archivar o borrar</string>
+    <string name="settingsAutoAdvanceListOfThreadsTitle">Lista de temas</string>
+    <string name="settingsAutoAdvanceNaturalThreadDescription">Repite la última acción realizada</string>
+    <string name="settingsAutoAdvanceNaturalThreadHint">Selecciona automáticamente la conversación siguiente o anterior, en función de tu última interacción.</string>
+    <string name="settingsAutoAdvanceNaturalThreadTitle">Repetir la última acción</string>
+    <string name="settingsAutoAdvancePreviousThreadDescription">Mostrar el hilo anterior al archivar o borrar</string>
```

Comment:
Try and replace all the usages of "hilo" that you added with "conversación" -> https://fr.pons.com/traduction/espagnol-fran%C3%A7ais/hilo